### ISSUE TITLE
Dynamic page size

### ISSUE THREAD

**Issue by petervelosy** [sproctor/compose-data-table#15]
There should be a way to make page size dynamic (so that it fills however much space it is given).

Unfortunately, the first parameter of `rememberPaginatedDataTableState` does not seem to be an "initial page size", but a "fixed page size" at the moment, as the page size neither shrinks nor expands according to the available space.

**Comment by sproctor** [comment]
Yeah, that would be a pretty cool feature. If you want to implement it, I'd be happy to merge it. It's not something I plan to use, so I probably won't implement it myself.
